test/misc.test.js

```diff
@@ -80,21 +78,14 @@ test("#85 Should handle state changing in constructors", done => {
             </span>
         )
     })
-    ReactDOM.render(<ParentWrapper />, testRoot, () => {
-        expect(testRoot.getElementsByTagName("span")[0].textContent).toBe("child:3 - parent:2")
-        a.set(5)
-        setTimeout(() => {
-            expect(testRoot.getElementsByTagName("span")[0].textContent).toBe("child:5 - parent:5")
-            a.set(7)
-            setTimeout(() => {
-                expect(testRoot.getElementsByTagName("span")[0].textContent).toBe(
-                    "child:7 - parent:7"
-                )
-                testRoot.parentNode.removeChild(testRoot)
-                done()
-            }, 10)
-        }, 10)
-    })
+    const { container } = render(<ParentWrapper />)
+    expect(container.getElementsByTagName("span")[0].textContent).toBe("child:3 - parent:3")
```

Comment:
I have to change this assertion from `parent:2` to `parent:3`.

Reply:
Um, why? I mean it does make sense to be like that, but why it would suddenly behave differently?

Also, can you please add `data-testid` attribute to `span` and then use `getByTestId` instead of this verbose thing?

Reply:
> Um, why? I mean it does make sense to be like that, but why it would suddenly behave differently?

It is the side-effect of using `act` from `react-dom/test-utils` in `@testing-library/react` `render`.

> Also, can you please add data-testid attribute to span and then use getByTestId instead of this verbose thing?

 Sure, thanks.

Reply:
> Also, can you please add data-testid attribute to span and then use getByTestId instead of this verbose thing?

I got rid of this query completely. I also added the `@testing-library/jest-dom` package to make the assertions more declarative.